spidermon/contrib/validation/jsonschema/formats.py

```diff
@@ -1,32 +1,22 @@
 from __future__ import absolute_import
-from jsonschema._format import FormatChecker, _draft_checkers
+from jsonschema._format import FormatChecker, _checks_drafts
 from jsonschema.compat import str_types
 
 from spidermon.contrib.validation.utils import is_valid_url, is_valid_email
-from six import iterkeys
 
 
+@_checks_drafts('url')
```

Comment:
This decorator overrides the custom format validator from jsonschema with a better one changing the `FormatChecker` instance that we use:
https://github.com/Julian/jsonschema/blob/v2.6.0/jsonschema/_format.py#L150

Until version 2.6.0, _draft_checkers[version] was a list (which was changed in 3.0.0), so the old way (concatenating two lists) wasn't valid for newer versions of jsonschema.

With this code applied, when we validate the following example, it will use our custom `is_url` function to validate the instance:
```
validate("localhost", {"format" : "url"})
validate(
   instance="some_url",
   schema={"format" : "url"}
)
```

Reply:
You mean it adds `url` format validator if `FormatChecker` is used? Because there's no `url` format validator in JSON schema.

Reply:
Yes. I know that `url` is not available, but it was included in spidermon before. I keep it for now.